README.md

```diff
@@ -62,8 +62,8 @@ insert into storage.buckets (id, name)
 Grab the project URL and anon key from the API settings and put them in a new .env.local file in the root directory as shown:
 
 ```
-NEXT_PUBLIC_SUPABASE_URL = "https://<project>.supabase.co";
-NEXT_PUBLIC_SUPABASE_ANON_KEY = "<your-anon-key>";
```

Comment:
removed ; so that new developers don't get errors if they copy-paste these two lines.

Reply:
ah thank you for doing this! i made this mistake myself and i now i know why